### ISSUE TITLE
Using with celery primatives?

### ISSUE THREAD

**Issue by jmerkow** [mback2k/django-celery-model#5]
Does this package with celery [primatives](http://docs.celeryproject.org/en/3.1/userguide/canvas.html#the-primitives), i.e. chords, groups, etc.?

P.S.
Btw, great package.

**Comment by mback2k** [comment]
Nope, not yet. Because the `model.apply_async` function fires of the task directly instead of returning a task signature that could be used with those primitives.

**Comment by jmerkow** [comment]
I would be interested in this, let me know if I can help.

**Comment by thenewguy** [comment]
I just happened to see this:
```
        result = group(signatures).apply_async()
        for child in result.children:
            ModelTaskMeta.objects.create(
                task_id=child.id,
                content_object=self,
            )
```
Easy way to relate groups if you have a reference to the instance you want to add
